waving_hands/gamemaster.py

```diff
@@ -61,15 +83,19 @@ def setup_game(self):
         # Wait for incoming connections and assign the sockets to the wizards.
         self.wait_for_connections()
 
-        self.customize()      # customize wizards. technically polish phase
+        if self.customize_wizards:
+            log.debug("Customizing Wizards")
+            self.customize()      # customize wizards. technically polish phase
 
         """@
         for wizard in self.wizards:
             wizard.hasted = True
             break
         """
 
-        self.pregame_flavor()
+        if self.pregame:
+            log.debug("Starting pregame")
+            self.pregame_flavor()
```

Comment:
Nice to have these optional for development testing purposes.

Reply:
Good idea! I remember I had commented it out when I was doing the original testing, but this is more flexible.